store/store.go

```diff
@@ -2015,14 +2022,14 @@ func isValidAction(action string) bool {
 }
 
 type snapActionJSON struct {
-	Action           string      `json:"action"`
-	InstanceKey      string      `json:"instance-key"`
-	Name             string      `json:"name,omitempty"`
-	SnapID           string      `json:"snap-id,omitempty"`
-	Channel          string      `json:"channel,omitempty"`
-	Revision         int         `json:"revision,omitempty"`
-	Epoch            *snap.Epoch `json:"epoch,omitempty"`
-	IgnoreValidation *bool       `json:"ignore-validation,omitempty"`
+	Action           string     `json:"action"`
+	InstanceKey      string     `json:"instance-key"`
+	Epoch            *epochJSON `json:"epoch"` // always send epoch, even if null
```

Comment:
Why is this significant?

Reply:
because the store won't send us an epoched snap unless it knows we can handle it, and we tell it we can handle it by always including it in the action

Reply:
This could be added as a comment in the code so that it doesn't get lost.

Reply:
Agree with @zyga, extending the comment would be useful.